spec/fixtures/config.yml

```diff
@@ -0,0 +1,2 @@
+# Dummy sample file for testing deposit of swift ingestor
+# testing file
```

Comment:
If these are testing file why keep the `config.yml` name? Wouldn't renaming them accordingly be more clearer? Like `example.txt` or something? Also does your tests need two "example files"? Why not use the same one twice?